code/src/components/MessageElement.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import moment from 'moment';
+
+const MessageElement = ({ input, onLikesIncrease }) => {
+    return (
+        <div>
+        <h4>{input.message}</h4>
+        <button onClick={() => onLikesIncrease(input._id)}>
+          {input.hearts}
+          ❤️
+        </button>
+        <p className="date">- {moment(input.createdAt).fromNow()}</p>
```

Comment:
Nice and clean 😃  if you would like to get the amount of hearts outside of the button you could move `{input.hearts}` to its own `<p></p>` tag for example. Then the heart will be a button and you can have the amount of hearts as a paragraph next to the button to follow the design as close as possible.